db_import/socrata_functions.R

```diff
@@ -33,10 +33,33 @@ create_sections_tables = function(
    
   if ( !'valuedate' %in% tolower(names(designation_tbl)) ) {
 
-    designation_tbl[, valuedate := as.character(NA) ]
+    designation_tbl[, valuedate := NA_character_ ]
 
   }
   
+  # col_type_chk = c(
```

Comment:
Please remove all the commented out code.  I think these conversions are implemented in a different function.